src/fund.rs

```diff
@@ -0,0 +1,93 @@
+use ethcore::{
+    self,
+    state::{CleanupMode, State},
+};
+use ethereum_types::{Address, H256, U256};
+use oasis_ethwasi_runtime_common::parity::NullBackend;
+extern crate chrono;
+use chrono::{DateTime, Datelike, NaiveDateTime, Utc};
+extern crate date_time;
+use byteorder::{BigEndian, ReadBytesExt};
+use date_time::month_tuple::MonthTuple;
+use std::{convert::TryInto, str::FromStr};
+
+fn timestamp_to_mt(timestamp: i64) -> MonthTuple {
```

Comment:
We need more comments to explain this function.

Reply:
done.